PlayCover/Views/MainView.swift

```diff
@@ -285,8 +287,12 @@ struct SplitViewAccessor: NSViewRepresentable {
             var sview = superview
 
             // Find split view through hierarchy
-            while sview != nil, !sview!.isKind(of: NSSplitView.self) {
-                sview = sview?.superview
+            if var sview = sview {
```

Comment:
Split view detection can be fixed by replacing this bit of code with
```swift
while let nsview = sview, !nsview.isKind(of: NSSplitView.self) {
    sview = sview?.superview
}
```
since `sview` needs to be checked for `nil` every loop instead of just the first.